layers/+tools/tmux/readme.md

```diff
@@ -0,0 +1,52 @@
+# tmux layer
+
+## Table of Contents
+
+<!-- vim-markdown-toc GFM -->
+* [Description](#description)
+* [Install](#install)
+* [Key Bindings](#key-bindings)
+
+<!-- vim-markdown-toc -->
+
+## Description
+
+This layer adds supports tmux operation.
+
+## Install
+
+To use this configuration layer, add it to your `~/.spacevim`.
+
+## Key Bindings
+
+Key Binding | Mode   | Description
+:---:       | :---:  | :---:
+<SPC>mb     | Normal | Sends ctrl+c to the associated pane.
+<SPC>mc     | Normal | Sends ctrl+l to the associated pane.
+<SPC>md     | Normal | fuzzy search tmux command and execute without argument
+<SPC>mf     | Normal | fuzzy search tmux copied buffer
+<SPC>mi     | Normal | fuzzy search tmux command and ask for argument
+<SPC>mk     | Normal | Prompts for input and sends it to the associated pane as
+<SPC>mm     | Normal | fuzzy search tmux  sessions
+<SPC>mn     | Normal | create a new tmux pane
+<SPC>mp     | Normal | Associate an already existing pane with tmuxify.
+<SPC>mq     | Normal | Closes the associated pane.
+<SPC>mr     | Normal | Run external command in a new tmux pane
+<SPC>ms     | Normal | Prompts for input and sends it to the associated pane.
+<SPC>mt     | Normal | Change the run command for the current filetype.
+<SPC>mw     | Normal | fuzzy search tmux  windows
+<C-h>       | Normal | TmuxNavigateRight
+<C-l>       | Normal | TmuxNavigateLeft
+<C-j>       | Normal | TmuxNavigateDown
+<C-k>       | Normal | TmuxNavigateUp
+
+Put following code to your tmux config:
```

Comment:
I think it's better to put the extra tmux config into install section in README.md. What do you think?